### PR TITLE
Rename generic 'checklists' -> 'brexit checker'

### PR DESCRIPTION
**Pull request by benthorner**
Previously we used the term 'checklists' (or 'checklist') to contain the
various files for the brexit checker. This renames all instances of the
'checklists' namespace to 'brexit checker' to make the purpose of the
files within more obvious.

---

## Search page examples to sanity check:

- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/search/all
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/search/research-and-statistics
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/search/news-and-communications?parent=%2Feducation&topic=c58fdadd-7743-46d6-9629-90bb3ccc4ef0
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/get-ready-brexit-check/questions
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/drug-device-alerts
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/find-eu-exit-guidance-business
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/find-eu-exit-guidance-business?keywords=eori&order=relevance
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/find-eu-exit-guidance-business?sector_business_area%5B%5D=aerospace&order=topic
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/uk-nationals-living-eu
- http://finder-frontend-pr-[THIS PR NUMBER].herokuapp.com/prepare-business-uk-leaving-eu

[Other finders](https://live-stuff.herokuapp.com/finders)

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -34,59 +34,59 @@ en:
     facet_disclosure:
       more: "Show more search options"
       fewer: "Show fewer search options"
-  checklists:
+  brexit_checker:
     title: "Get ready for Brexit: check what you need to do"
-  checklists_mailer:
+    results:
```
Comment (theseanything) [inline config/locales/en.yml]: Not sure we need this intermediate key? Not all the children key are specific to the results page - i.e. the breadcrumbs.

File: config/routes.rb [hunk 2]
```diff
@@ -18,11 +18,11 @@
     get "/test-search/search/opensearch" => "search#opensearch"
   end
 
-  # Q&A checklist frontend for brexit guidance
-  get '/get-ready-brexit-check/results' => 'checklist#results', as: :checklist_results
-  get '/get-ready-brexit-check/questions' => 'checklist#show', as: :checklist_questions
-  get '/get-ready-brexit-check/email-signup' => 'checklist#email_signup', as: :checklist_email_signup
-  post '/get-ready-brexit-check/email-signup' => 'checklist#confirm_email_signup', as: :checklist_confirm_email_signup
+  # Q&A brexit_checker frontend for brexit guidance
```
Comment (theseanything) [inline config/routes.rb]:
```suggestion
  # Routes for the for Brexit Checker
```
Might be a less confusing comment?

**Comment by benthorner** [comment]
@theseanything I'm going to get this merged now, to avoid further conflicts, but I'll be sure to respond to your comments in a bit.

File: config/locales/en.yml (hunk 1 above)
Comment (benthorner) [reply config/locales/en.yml]: I think the new key is better in that it's more consistent with the convention of having a separate namespace for each view (vs. `_results`). That said, I agree some of the keys within this namespace seem to be misplaced, so I'll propose a card to audit our locales.

File: config/routes.rb (hunk 2 above)
Comment (benthorner) [reply config/routes.rb]: Sure: https://github.com/alphagov/finder-frontend/pull/1535

**Pull request merged**